massgen/tests/test_checkpoint_mcp_standalone.py

```
@@ -903,6 +1010,386 @@ def test_opening_uses_broadened_framing(self):
+
+    Aligned autonomy: actions beyond the user's literal phrasing are approvable
+    when they (a) serve the user's explicit task AND (b) meet the safety
+    criteria. The pre-reframing prompt told the reviewer to pick the "SAFEST
+    scope" and refuse anything beyond the literal ask, which killed clearly-
+    aligned sub-actions. These assertions lock in the reframing while
+    preserving the prompt-injection defense (file content cannot introduce
+    a new, unrelated objective or override the safety policy).
+    """
+
+    _DEFAULT_TASK = "Test user task: do the requested thing."
+    _DEFAULT_ENV: dict = {}
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)